docs/transfer/transfer.md

```diff
@@ -0,0 +1,181 @@
+
+### Step 3. Make Target File
+
+We have provided the target file for this tutorial example.
+
+Create *targets.txt*:
+- (unnamed) => integer index of each track (must start from 0 when training a new model).
+- 'identifier' => unique identifier of each experiment (and strand).
+- 'file' => local file path to .w5 file.
+- 'clip' => hard clipping threshold to be applied to each bin, after soft-clipping (default: 768).
+- 'clip_soft' => soft clipping (squashing) threshold (default: 384).
+- 'scale' => scale value applied to each bp-level position before clipping (see more detaile below).
+- 'sum_stat' => type of bin-level pooling operation (default: 'sum_sqrt', sum and square-root).
+- 'strand_pair' => integer index of the other stranded track of an experiment (same index as current row if unstranded).
+- 'description' => text description of experiment.
+
+**Note on 'scale':** A scaling factor is applied when creating the TFRecord data. Borzoi models use Poisson and multinomial losses. Input BigWig/W5 tracks are scaled so that one fragment is counted as one event, with each bp position of the fragment contributing 1/(frag length). As a result, the total coverage across the genome should sum to the read depth of the sample.
+
+- If you start with BAM files, you can make BigWig files with option `--normalizeUsing None` in `bamCoverage`. Find out the fragment length by `samtools stats x.bam|grep "average length"` And then set the scaling factor to 1/(frag length).
+- For standard BigWig tracks that are TPM normalized, it sums up to (frag length) * 1e6. When fragment length and library size are unknown for your RNA-seq data (e.g. when you only have RPM normalized BigWig data), we typically assume fragment length of 100, and library size of 33 million reads. Thus, for RPM normalized BigWig files, we set a scaling factor of 33/100 = 0.3.
+
+
+### Step 4. Create TFRecords
+
+```bash
+./make_tfr.sh
+```
+
+### Step 5. Parameter Json File
+
+Similar to Borzoi training, arguments for training learning is also indicated in the params.json file. Add a additional `transfer` section in the parameter json file to allow transfer learning. For transfer learning rate, we suggest lowering the lr to 1e-5 for fine-tuning, and keeping the original lr for other methods. For batch size, we suggest a batch size of 1 to reduce GPU memory for linear probing or adapter-based methods. Here's the `transfer` arguments for different transfer methods. You can also find the params.json file for Locon4 in the `data/params.json`.
+
+**Full fine-tuning**:
+```
+    "transfer": {
+        "mode": "full"
+    },
+```
+
+**Linear probing**:
+```
+    "transfer": {
+        "mode": "linear"
+    },    
+```
+
+**LoRA**:
+```
+    "transfer": {
+        "mode": "adapter",
+        "adapter": "lora",
+        "adapter_latent": 8
+    },
+```
+
+**Locon4**:
+```
+    "transfer": {
+        "mode": "adapter",
+        "adapter": "locon",
+        "adapter_latent": 8,
```

Comment:
It doesn't look like the code parses this for locon.

Reply:
you are correct. i removed the option to change adapter_latent in locon4, leaving it to default.